pkg/tecdsa/marshaling.go

```diff
@@ -0,0 +1,167 @@
+package tecdsa
+
+import (
+	"fmt"
+	"github.com/binance-chain/tss-lib/crypto"
+	"github.com/binance-chain/tss-lib/crypto/paillier"
+	"github.com/binance-chain/tss-lib/ecdsa/keygen"
+	"github.com/keep-network/keep-core/pkg/tecdsa/gen/pb"
+	"math/big"
+)
+
+// ErrIncompatiblePublicKey indicates that the given public key is not
+// compatible with the tECDSA, i.e. uses a different elliptic curve.
+// Such a key cannot be marshalled and unmarshalled.
+var ErrIncompatiblePublicKey = fmt.Errorf(
```

Comment:
Do we need to export it out of `tecdsa`?

Reply:
My reasoning: `tecdsa.PrivateKeyShare` is a public type so as their `Marshal` function. If that function returns a pre-defined error, it would be good to expose that error too.

Reply:
I missed that `tecdsa.PrivateKeyShare` is marshalled from `tbtc` package. It all makes sense!